flask_security/core.py

```diff
@@ -88,13 +89,14 @@
     'CHANGE_SALT': 'change-salt',
     'REMEMBER_SALT': 'remember-salt',
     'DEFAULT_REMEMBER_ME': False,
-    'DEFAULT_HTTP_AUTH_REALM': 'Login Required',
-    'EMAIL_SUBJECT_REGISTER': 'Welcome',
-    'EMAIL_SUBJECT_CONFIRM': 'Please confirm your email',
-    'EMAIL_SUBJECT_PASSWORDLESS': 'Login instructions',
-    'EMAIL_SUBJECT_PASSWORD_NOTICE': 'Your password has been reset',
-    'EMAIL_SUBJECT_PASSWORD_CHANGE_NOTICE': 'Your password has been changed',
-    'EMAIL_SUBJECT_PASSWORD_RESET': 'Password reset instructions',
+    'DEFAULT_HTTP_AUTH_REALM': _('Login Required'),
```

Comment:
That is true. I guess this is ultimately a use-case philosophical discussion. People following the Flask Mega-Tutorial (myself included) wrap all configuration strings to be translated in lazy_gettext. This would work as well, and the translated files would end up in that person .mo files.

I can spend some time on an alternative solution if you're sure. Perhaps adding a callback somewhere. But I think its more complexity for no clear benefit.